### PR TITLE
Unmute & Mute

### PR DESCRIPTION
**Pull request by aliyss**

### CODE REVIEW

File: src/structures/Chat.js [hunk 1]
```diff
@@ -57,6 +57,12 @@ class Chat extends Base {
          */
         this.archived = data.archive;
 
+        /**
+         * Indicates if the Chat is muted
+         * @type {boolean}
+         */
+        this.isMuted = data.mute.isMuted;
```
Comment (Aliyss) [inline src/structures/Chat.js]: Not sure if this works. But in the browser `mute` is definitely an object.
I forgot if `data` is serialized or not.

File: src/structures/Chat.js [hunk 2]
```diff
@@ -112,6 +118,21 @@ class Chat extends Base {
         return this.client.unarchiveChat(this.id._serialized);
     }
 
+    /**
+     * mutes this chat
+     * @param {Number} timestamp
```
Comment (pedroslopez) [inline src/structures/Chat.js]: Can you describe what this time stamp refers to?

Comment (Aliyss) [reply src/structures/Chat.js]: Timestamp is the exparation time

Comment (pedroslopez) [reply src/structures/Chat.js]: Oh, can it really be any arbitrary date not in those intervals?

Also, maybe this should be a JS date and do the time stamp conversion internally for convenience.

**Comment by Aliyss** [comment]
@pedroslopez

Mind testing it for me. I am not even sure it works correctly. And can't really test on my current system.

**Comment by Aliyss** [comment]
After testing this does not work as intended. I will try to fix it.

**Comment by Aliyss** [comment]
Deprecated

**Pull request closed without merging**